src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -25,92 +48,1243 @@
     public String groupByTeam = null;
     public String showStats = null;
     public int startIndex;
+    private boolean shouldCollapsed;
+    
+    private FieldValidator validator = new FieldValidator();
+
 
     // used for html table ajax loading
     public String courseId = null;
     public String feedbackSessionName = null;
     public String ajaxStatus = null;
     public String sessionResultsHtmlTableAsString = null;
+    
 
+    // TODO multiple page data classes for each view type
+    
+    // for question view
+    List<InstructorResultsQuestionTable> questionPanels;
+    // for giver > question > recipient, recipient > question > giver...
+    LinkedHashMap<String, InstructorFeedbackResultsSectionPanel> sectionPanels;
```

Comment:
The original logic, and therefore the current logic here, relies specifically on the iteration order of the map, which is implementation specific. I think using <code>Map</code> is a little misleading